awx/playbooks/action_plugins/insights.py

```diff
@@ -36,7 +37,9 @@ def write_version(self, proj_path, etag):
             f.write(etag)
 
     def _obtain_auth_token(self, oidc_endpoint, client_id, client_secret):
-        main_url = urljoin(oidc_endpoint, '/.well-known/openid-configuration')
+        if oidc_endpoint.endswith('/'):
```

Comment:
with the below line change, we no longer need this conditional, rstrip will take off the `/` if it is there, otherwise it does nothing